test/cucumber/ParkinSpaceSuggestion.feature

```diff
@@ -0,0 +1,48 @@
+Feature: parking space suggestion
+  As driver entering a parking lot
+  I want to find the best parking space
```

Comment:
Acho que nessa feature, "melhor" vaga não é um termo muito bem definido porque não tem algo que defina a melhor vaga, como por exemplo na feature de preferência de destino. Talvez o ideal fosse deixar aqui apenas "I want to find a parking space".